v0/destinations/mp/data/MPEventPropertiesConfig.json

```diff
@@ -150,5 +162,13 @@
   {
     "sourceKeys": "properties.googlePlayServices",
     "destKey": "$google_play_services"
+  },
+  {
+    "sourceKeys": "messageId",
+    "destKey": "$insert_id"
+  },
+  {
+    "sourceKeys": "properties.mpApiEndpoint",
+    "destKey": "$mp_api_endpoint"
```

Comment:
This value should be set to host header ref:
Mp Api Endpoint (raw property name: $mp_api_endpoint)
This is the value of the [Host Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Host) based on the request to our ingestion service. This value will commonly be api.mixpanel.com. However, it is useful to confirm where your data was ingested if you partake in our [EU Data Residency](https://help.mixpanel.com/hc/en-us/articles/360039135652-Data-Residency-in-EU) or are using a custom proxy.

We don't need to set this from property, lets omit this field

Reply:
removed